mkdocs.yml

```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
 nav:
   - Getting started:
     - Introducing episb: README.md
-  - How-to guides:
+  - HUB user guides:
```

Comment:
no need to capitalize HUB

Reply:
I think I will just do away with the word "hub" as it is. See new commit.